docs/source/conf.py

```diff
@@ -25,7 +25,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'sam'
-copyright = '2018-2021, Royal HaskoningDHV'
+copyright = '2018-2022, Royal HaskoningDHV'
 author = 'Fenno, Daan, Rutger, Arjan, Loes, Tim, Sebastiaan'
```

Comment:
Author = Royal HaskoningDHV?

Reply:
Yeah, I can change all the authors. Should also change it in setup.py and readme then